app/renderer/js/main.js

```diff
@@ -147,6 +149,20 @@ class ServerManagerView {
 		}
 	}
 
+	initEnterpriseSettings() {
+		const filepath = ConfigUtil.getConfigItem('enterpriseFilepath', null);
+		if (filepath) {
+			fs.readFile(filepath, (err, data) => {
+				if (err) {
+					logger.reportSentry(err);
```

Comment:
This is wrong. If the config file doesn't exist it will report that to sentry and spam it with these reports. It is ok for the config file to not exist.